docs/malware-scanner/README.md

```diff
@@ -1 +1,9 @@
-## *Currently empty, check back later or contribute yourself*
\ No newline at end of file
+## General Malware Scanner
+[IsThisARat?](https://isthisarat.com/)
+
+Is an Rat finder powerd by AI
```

Comment:
I don't think it's wise to recommend relatively unknown websites to use. MMPA is already working on its own malware scanner, that can be linked when its in a working state.

Reply:
I agree. Obfuscation [as detected here](https://isthisarat.com/score/d82f176a5877dd941ad2f8586becd00ef0bed75be75b7f029f6b649ec67f4a45) is unfortunately very common among Minecraft plugins. Like the other plugin, it's also proprietary.

In my personal opinion, this should not be recommended by this project.

Reply:
> I agree. Obfuscation [as detected here](https://isthisarat.com/score/d82f176a5877dd941ad2f8586becd00ef0bed75be75b7f029f6b649ec67f4a45) is unfortunately very common among Minecraft plugins. Like the other plugin, it's also proprietary.
> 
> In my personal opinion, this should not be recommended by this project.

Agreed 👍